XRDXRFutils/notebook_utils.py

```diff
@@ -163,6 +163,19 @@ def shorten_string(s, length_max):
     return s_short
 
 
+def read_parameters_calibration_xrf(filename):
```

Comment:
What are these files that we are reading?

Reply:
I attach an example of these files. They are simple files that contain the parameters of calibration for XRF spectra. In particular, they contain a, b, c for the calibration of type `a x^2 + b x + c`.
Paolo and Eva need to be able to calibrate any XRF spectrum taking that file as the source of calibration.

[calib_XRF_2nd.calib.zip](https://github.com/zpreisler/XRDXRFutils/files/10881333/calib_XRF_2nd.calib.zip)